site/es/blog/new-in-devtools-92/index.md

```diff
@@ -0,0 +1,224 @@
+
+Puedes ver todos los cambios de Lighthouse en sus [nota de lanzamiento](https://github.com/GoogleChrome/lighthouse/releases/tag/v7.5.0).
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/4cef9324af0e4560421beb138313458d5ae6fb0b #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d17898e62fe19b9f47b25f1568b57fce951c6d10 #}
+
+Chromium issue: [772558](https://crbug.com/772558)
+
+## "Restart frame" queda obsoleto en el menú contextual de la pila de llamada {: #restart-frame }
+
+La opción **Restart frame** queda obsoleta. Esta característica requiere más desarrollo para que funcione correctamente ya que actualmente está rota y falla frecuentemente.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Alvnt4FkoEFoP0SkdKgi.png", alt="El menú contextual Restart frame ha quedado obsoleto", width="800", height="486" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/4494098b6840f608347c1edf3c048691056eada4 #}
+
+Chromium issue: [1203606](https://crbug.com/1203606)
+
+## Monitor de protocolo [Experimental] {: #protocol-monitor }
+
+{% Aside %}
+
+Para activar este experimento, activa la casilla **Protocol Monitor** en **Settings > Experiments**.
+
+{% endAside %}
+
+Chrome DevTools usa el [Chrome DevTools Protocol (CDP)](https://chromedevtools.github.io/devtools-protocol/) para instrumentalizar, inspeccionar, depurar y analizar el comportamiento de Chrome. El **monitor de Protocolo** te provee una forma de ver todas las peticiones CDP y las respuestas realizadas por DevTools.
+
+Dos nuevas funcionalidades han sido añadidas para facilitar las pruebas de CDP:
+
+- Un nuevo botón **Save** que te permite guardar las respuestas CDP en un archivo JSON.
+- Un nuevo campo que te permite enviar un comando CDP directamente.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/mRVrHC9WEet7cwA7QAeV.png", alt="Protocol monitor", width="800", height="496" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/40fcb9a9aae81ac1df2c19dee467ab3a4cf4088b #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/006e20c4226a7b2e5fde4026191b1eaf05bf8b8e #}
+
+Chromium issues: [1204004](https://crbug.com/1204004), [1204466](https://crbug.com/1204466)
+
+## Grabadora de Puppeteer [Experimental] {: #puppeteer-recorder }
+
+{% Aside %}
+
+Para activar este experimento, activa la casilla **Recorder** en **Settings > Experiments**.
+
+{% endAside %}
+
+[La grabadora de Puppeteer](https://developer.chrome.com/blog/new-in-devtools-89/#record) ahora genera una lista de pasos basada en tu interacción con el navegador, donde antes las DevTools generaba directamente un script de Puppeteer. Un nuevo botón **Export** ha sido añadido que te permite exportar los pasos como un script de Puppeteer.
```

Comment:
[La grabadora de Puppeteer](https://developer.chrome.com/blog/new-in-devtools-89/#record) ahora genera una lista de pasos basada en tu interacción con el navegador, donde antes DevTools generaba directamente un script de Puppeteer. Un nuevo botón **Export** ha sido añadido que te permite exportar los pasos como un script de Puppeteer.